### DISCUSSION TITLE
Users with F3 license

### DISCUSSION THREAD

**Discussion by MarianDanisek** [Set-OutlookSignatures/Set-OutlookSignatures#68] [General]
Is it possible to generate email signature for users with ONLY M365 F3 licenses - only web access to Outlook, Office suite ?

**Comment by GruberMarkus** [comment]
Yes, Set-OutlookSignatures supports deployments with Outlook Web (OWA) only.

The documentation included in the download contains all the information you need to implement this feature. If you look for further assistance, please contact me regarding commercial support.

**Comment by MichalPrepilka** [comment]
If I can I want to join in conversation. How is it possible to create .docx or .html or .rtf (or some kind of signature applicable for OWA) signature from template when using M365 F3 licence for OWA users? F3 licence is without Microsoft Word application in local computer (only cloud/browser version ). You are using -ComObject Word.Application and WORD is must have for this module. So is there some kind of workaround? 
Signature is applicable for OWA but I have to create it first.

**Comment by GruberMarkus** [comment]
@MichalPrepilka:
- Word will not be used at all when you use a HTM template and set the parameters 'CreateRTFSignatures' and 'CreateTXTSignatures' to false.
- The script download contains sample code and extensive documentation showing how to create and set Outlook Web signatures without any end user interaction.
- If you look for further assistance, please contact me regarding commercial support.

**Reply by MichalPrepilka** [reply]
I will try it. Thank you

**Comment by GruberMarkus** [comment]
Update: Starting with v4.0.0, not only the classic single signature can be deployed to Outlook Web for F3 users (which can not run Set-OutlookSignatures themselves), but also the new roaming signatures.
